src/NHSDPublicBrowseAcceptanceTests.TestData/Utils/Queries.cs

```diff
@@ -76,34 +76,40 @@ from Solution
 
         public const string GetSolutionCountForCapability =
-            "SELECT COUNT(*) FROM Solution s INNER JOIN SolutionCapability sc on sc.SolutionId=s.Id INNER JOIN Capability c on sc.CapabilityId = c.Id WHERE c.Name=@CapabilityName";
+            "SELECT COUNT(*) FROM Solution s INNER JOIN SolutionCapability sc on sc.SolutionId=s.Id INNER JOIN Capability c on sc.CapabilityId = c.Id WHERE c.Name=@CapabilityName;";
 
         public const string CreateNewCatalogueItem =
-            "INSERT INTO CatalogueItem (CatalogueItemId, Name, CatalogueItemTypeId, SupplierId, PublishedStatusId, Created) values (@CatalogueItemId, @Name, @CatalogueItemTypeId, @SupplierId, @PublishedStatusId, @Created)";
+            "INSERT INTO CatalogueItem (CatalogueItemId, Name, CatalogueItemTypeId, SupplierId, PublishedStatusId, Created) values (@CatalogueItemId, @Name, @CatalogueItemTypeId, @SupplierId, @PublishedStatusId, @Created);";
 
-        public const string GetCatalogueItem = "SELECT * from [dbo].[CatalogueItem] WHERE CatalogueItemId=@CatalogueItemId";
+        public const string GetCatalogueItem = "SELECT * from [dbo].[CatalogueItem] WHERE CatalogueItemId=@CatalogueItemId;";
 
         public const string UpdateCatalogueItem =
-            "UPDATE CatalogueItem SET Name=@Name, CatalogueItemTypeId=@CatalogueItemTypeId, SupplierId=@SupplierId, PublishedStatusId=@publishedStatusId, Created=@Created WHERE CatalogueItemId=@CatalogueItemId";
+            "UPDATE CatalogueItem SET Name=@Name, CatalogueItemTypeId=@CatalogueItemTypeId, SupplierId=@SupplierId, PublishedStatusId=@publishedStatusId, Created=@Created WHERE CatalogueItemId=@CatalogueItemId;";
 
-        public const string DeleteCatalogueItem = "DELETE FROM CatalogueItem WHERE CatalogueItemId=@CatalogueItemId";
-        public const string GetAllCatalogueItems = "SELECT * FROM [dbo].[CatalogueItem]";
+        public const string DeleteCatalogueItem = "DELETE FROM CatalogueItem WHERE CatalogueItemId=@CatalogueItemId;";
+        public const string GetAllCatalogueItems = "SELECT * FROM [dbo].[CatalogueItem];";
 
-        internal const string GetLastUpdated = "SELECT LastUpdated FROM @table WHERE @whereKey=@whereValue";
+        public const string GetFrameworkSolutionCount = @"SELECT COUNT(*)
+                                                              FROM FrameworkSolutions AS fs
+                                                              INNER JOIN CatalogueItem AS ci ON ci.CatalogueItemId = fs.SolutionId
+                                                              WHERE FrameworkId IN(SELECT TOP(1) Id FROM Framework WHERE ShortName = @frameworkName)
+                                                              AND ci.PublishedStatusId IN (3, 4);";
+
+        internal const string GetLastUpdated = "SELECT LastUpdated FROM @table WHERE @whereKey=@whereValue;";
```

Comment:
Terminating semicolons – like it! 😃 

There are a few other improvements that you could make to the SQL formatting but non-essential, e.g.

1. Space after comma.
2. Square brackets only where required.
3. Use schema name when referencing tables.
4. Spaces around operators.

Reply:
I'll get around to those as and when I look back to reformatting these queries